Google.Api.Generator.Tests/ProtoTests/BasicClient.cs

```diff
@@ -59,5 +61,55 @@ public abstract partial class BasicClient
             "scope1",
             "scope2",
         });
+
+        private static readonly gaxgrpc::ChannelPool s_channelPool = new gaxgrpc::ChannelPool(DefaultScopes);
+
+        /// <summary>Asynchronously creates a <see cref="BasicClient"/>, applying defaults for all unspecified settings, and creating a channel connecting to the given endpoint with application default credentials where necessary. See the example for how to use custom credentials.</summary>
+        /// <example>This sample shows how to create a client using default credentials:<code>using Google.Cloud.Vision.V1;...// When running on Google Cloud Platform this will use the project Compute Credential.// Or set the GOOGLE_APPLICATION_CREDENTIALS environment variable to the path of a JSON// credential file to use that credential.ImageAnnotatorClient client = await ImageAnnotatorClient.CreateAsync();</code>This sample shows how to create a client using credentials loaded from a JSON file:<code>using Google.Cloud.Vision.V1;using Google.Apis.Auth.OAuth2;using Grpc.Auth;using Grpc.Core;...GoogleCredential cred = GoogleCredential.FromFile("/path/to/credentials.json");Channel channel = new Channel(    ImageAnnotatorClient.DefaultEndpoint.Host, ImageAnnotatorClient.DefaultEndpoint.Port, cred.ToChannelCredentials());ImageAnnotatorClient client = ImageAnnotatorClient.Create(channel);...// Shutdown the channel when it is no longer required.await channel.ShutdownAsync();</code></example>
```

Comment:
It looks like the line breaks aren't working.

I'll wait until that's sorted out before reviewing.

Reply:
(Ah, maybe that's what you meant by the XMLDoc formatting being terrible. Okay, will keep reviewing.)

Reply:
I'm working on a formatter for the XMLDoc (and the code) in parallel; it's not entirely trivial.
So I'd rather merge this as-is. There is a TODO in the code to address this.